install/docker/console-db-es-ls-kibana/config.properties

```diff
@@ -0,0 +1,122 @@
+# ---------------------------------------------------------------------------
+#  This is an example properties file that can be used when configuring CWS
+#  for the first time.
+#  
+#  To use:
+#
+#    1)  Fill in the correct values where you see [YourXXX] below, for example
+#        [YourHostname].
+#        The hard-coded values are suggested to be used, but can be changed if
+#        desired.
+# 
+#    2)  Run CWS configuration program with this properties file:
+#        ./configure.sh config.properties
+#
+# ---------------------------------------------------------------------------
+
+cws_ldap_url_default=ldaps://localhost:636
+ldap_identity_plugin_class=org.camunda.bpm.identity.impl.ldap.plugin.LdapIdentityProviderPlugin
+ldap_security_filter_class=jpl.cws.core.web.CwsLdapSecurityFilter
+camunda_security_filter_class=jpl.cws.core.web.CwsCamundaSecurityFilter
+auto_accept_config=y
+startup_autoregister_process_defs=false
+
+# This is the hostname of the machine you are installing CWS (could be a Worker,
+# a Console, or both) on. The name (or less preferably IP address) you use here
+# should be accessible by all other components inthe CWS installation.
+hostname=cws-console
+
+# This is the type of installation you are performing
+# (1 = Console and Worker, 2 = Console Only, 3 = Worker Only)
+install_type=1
+
+database_type=mariadb
+
+# This is the hostname where the database is installed.
+# This hostname must be accessible by all Workers and Console
+database_host=db
+
+database_port=3306
+
+# The name of the database schema.
+database_name=cws
+
+# The name of the user that can access, and has CRUD permission to the database
+database_username=root
```

Comment:
Consider parameterizing this.

Reply:
All of this is just for a local instance of cws.  It is basically a sample of what to use, but should be changed by the user.  It's not a security risk because it is a local sample setup.